### DISCUSSION TITLE
Do y'all have any ideas for Drop?

### DISCUSSION THREAD

**Discussion by EnZon3** [EnZon3/Drop#11] [Ideas]
Besides the general roadmap, I want to poll for some ideas.. because I want to make Drop as good as possible for communities.

**Comment by EnZon3** [comment]
Anyone?

**Comment by MilesWK** [comment]
Python support. And with python support, add in hosting options
(PLEEASSE MAKE THEM FREE.) and also a nice console output log. I just want
python!

**Reply by EnZon3** [reply]
I never really intended for Drop to be a hosting solution, moreso just a place to share your project like Product Hunt. Although, I'll keep that in mind! (Of course, i'll try my best to make it free.)

**Comment by Fairies-Feast** [comment]
How will Drop work anyways? Do you share Github Repos or is it just ReplTalk?

**Reply by EnZon3** [reply]
In a nutshell, it's a place to share your projects. So, you can link your project's url or GitHub repository to the project, and the algorithm (planned not implemented yet) will recommend it to people. It's basically the community aspect of Replit, or a more casual Product Hunt. Adding hosting isn't out of the question though.

I wanted to give the freedom of using any hosting provider to users so that's why I initially didn't include a hosting solution. Now that y'all mention it, I'm thinking of adding one; maybe for small python projects or whatnot.

**Reply by Fairies-Feast** [reply]
@EnZon3 If you are making hosting, you can try Docker (it has SDKs for node + python), but Docker does not work on Vercel.

**Reply by EnZon3** [reply]
I'll try to figure something out.

**Comment by MilesWK** [comment]
Cool!

**Comment by SalladShooter** [comment]
Do you have a logo currently? If not, I’ve made a mockup, feel free to give feedback, and feel free to use it. I made the logo easy to recognize (from the text), and since you aren’t a big company a single icon or letter wouldn’t do, so I used an approach similar to my project phyal.
![D-removebg-preview](https://github.com/user-attachments/assets/8de86852-ed7a-4a3f-8c1e-33701fd8e5b7)

**Reply by EnZon3** [reply]
Good start but not what i'm looking for. Fun fact where the name "Drop" came from was from one of the definitions of the word drop, e.g. in the sentence "he just dropped a new website"

**Reply by SalladShooter** [reply]
Do you have anything in mind? If so, I could probably make it for you.

**Reply by EnZon3** [reply]
I don't really know.
